app/change_set_persisters/hyrax/collection_change_set_persister.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literals: true
+
+module Hyrax
+  class CollectionChangeSetPersister < ChangeSetPersister
+    after_save :process_member_changes
+
+    def process_member_changes(change_set:, resource:)
+      case change_set.members
+      when 'remove'
+        remove_members_from_collection(change_set: change_set, resource: resource)
+      when 'move'
+        move_members_between_collections(change_set: change_set, resource: resource)
+      else
+        add_members_to_collection(change_set: change_set, resource: resource)
+      end
+    end
+
+    private
+
+      def remove_members_from_collection(change_set:, resource:)
+        Array(change_set.batch).each do |member_id|
+          member = find_resource(member_id)
+          member.member_of_collection_ids.delete resource.id
+          persister.save(resource: member)
+        end
+      end
+
+      def add_members_to_collection(change_set:, resource:)
+        Array(change_set.batch).each do |member_id|
+          member = find_resource(member_id)
+          member.member_of_collection_ids << resource.id
```

Comment:
Is `<<` okay here or should it be `+=`?

Reply:
I'm not seeing any frozen array modification error, so I guess it's okay.

Reply:
I think ChangeSets handle defaults differently.